src/openforms/tests/e2e/test_admin_js.py

```diff
@@ -0,0 +1,28 @@
+from django.urls import reverse
+
+from furl import furl
+from playwright.async_api import expect
+
+from .base import E2ETestCase, browser_page, create_superuser
+
+
+class AdminJSTests(E2ETestCase):
+    async def test_admin_pages_with_js_bundle_dont_crash(self):
```

Comment:
For some reason, this test fails locally for me with the chromium/webkit driver (I am not able to run with firefox at all :thinking:):
```
Failure
Traceback (most recent call last):
  File "/home/silvia/repositories/open-forms/src/openforms/tests/e2e/test_admin_js.py", line 28, in test_admin_pages_with_js_bundle_dont_crash
    await expect(marker).to_have_count(1, timeout=500)
  File "/home/silvia/.pyenv/versions/open-forms/lib/python3.10/site-packages/playwright/async_api/_generated.py", line 18998, in to_have_count
    await self._impl_obj.to_have_count(count=count, timeout=timeout)
  File "/home/silvia/.pyenv/versions/open-forms/lib/python3.10/site-packages/playwright/_impl/_assertions.py", line 261, in to_have_count
    await self._expect_impl(
  File "/home/silvia/.pyenv/versions/open-forms/lib/python3.10/site-packages/playwright/_impl/_assertions.py", line 55, in _expect_impl
    raise AssertionError(
AssertionError: Locator expected to have count '1'
Actual value: 0 
Call log:
LocatorAssertions.to_have_count with timeout 500ms
waiting for get_by_test_id("e2e-test-id")
  locator resolved to 0 elements
  unexpected value "0"
waiting for get_by_test_id("e2e-test-id")
  locator resolved to 0 elements
  unexpected value "0"
  locator resolved to 0 elements
  unexpected value "0"
```

Have you had this before?

Reply:
Ahh maybe I forgot to do that, I shall try again!